src/index.js

```diff
@@ -13,6 +13,16 @@ const app = express();
 
 dotenv.config();
 
+app.use(cors());
+app.use((req, res, next) => {
```

Comment:
@urbanishimwe  great job but also the second middleware is also not required when you're using CORS middleware package basically you need to add only `app.use(cors());`